extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/main/java/io/quarkus/resteasy/reactive/server/deployment/ResteasyReactiveProcessor.java

```diff
@@ -24,6 +26,7 @@
 import javax.ws.rs.ext.MessageBodyReader;
 import javax.ws.rs.ext.MessageBodyWriter;
 
+import org.apache.commons.lang3.ArrayUtils;
```

Comment:
Not sure if depending on commons-lang is necessary here. Perhaps it's easier to just copy/paste the `ArrayUtils.isEmpty` method to here to avoid the dependency